### ISSUE TITLE
Analytic events and baseline tracking for homepage

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4930]
**Description:**
To define elements that should be tracked in our homepage (IA refresh mockup). Also, find commonalities with current page and set baseline metrics so we can compare before/after.

Relevant documents:
[Trackmaster](https://docs.google.com/document/d/1lplwg9kz4CJ7N5y0Noq84tmjXprqsKP_1nqtKkeCp28/edit?usp=sharing)
[Analytics site events (created by Matthew)](https://docs.google.com/spreadsheets/d/1x-CP_HYaAs3oRv2cjbSFu8w5Z938E4cyHQ3hvh17c6c/edit?usp=sharing)

**Comment by taisdesouzalessa** [comment]
**Here are the recommendations for things we should track from homepage:**
• [GA - Tracking spreadsheet](https://docs.google.com/spreadsheets/d/1GFmZXZu6R9aUygeBLDONODt7-GwwwIz7ioNP70o3mfY/edit?usp=sharing)

I also updated the [Analytics Site Events spreadsheet ](https://docs.google.com/spreadsheets/d/1x-CP_HYaAs3oRv2cjbSFu8w5Z938E4cyHQ3hvh17c6c/edit?usp=sharing)(lines 30-35) - I did my best to replicate the model Matthew had and asked for feedback from Mavis.

Do we need any approval of this work @kristinashu?

Once I have the answer, the next step is to file implementation tickets to those events (blocked by having the homepage ready).
